### PR TITLE
review code of torrentfs

### PR DESCRIPTION
**Pull request by ShadowErii**
Is it necessary to add test for these simple funcs like this to increse the coverage?

### CODE REVIEW

**Comment by ucwong** [comment]
Yes the more the better

File: compress/compress_test.go [hunk 1]
```diff
@@ -2,10 +2,29 @@ package compress
...
 import (
 	"fmt"
+	clog "github.com/CortexFoundation/CortexTheseus/log"
+	"github.com/golang/snappy"
 	"log"
 	"testing"
+	"time"
 )
 
+func SnappyEncode(data []byte) ([]byte, error) {
```
Comment (ucwong) [inline compress/compress_test.go:12]: why move these methods here?

File: compress/compress.go [hunk 2]
```diff
@@ -3,15 +3,14 @@ package compress
 import (
 	"bytes"
 	"compress/gzip"
-	"github.com/CortexFoundation/CortexTheseus/log"
-	"github.com/golang/snappy"
+	clog "github.com/CortexFoundation/CortexTheseus/log"
```
Comment (ucwong) [inline compress/compress.go:6]: Is it necessary to use a cname?

Comment (ShadowErii) [reply compress/compress.go:6]: fotget to change back

File: compress/compress_test.go (hunk 1 above)
Comment (ShadowErii) [reply compress/compress_test.go:12]: it seems that was just used for test

Comment (ucwong) [reply compress/compress_test.go:12]: > it seems that was just used for test

Oh no , it is the same with zip, another compress method but not used now.

Comment (ShadowErii) [reply compress/compress_test.go:12]: it seems that was only used in test

Comment (ucwong) [reply compress/compress_test.go:12]: https://github.com/golang/snappy

Comment (ucwong) [reply compress/compress_test.go:12]: You can just add some comment above them to fix the linter

Comment (ShadowErii) [reply compress/compress_test.go:12]: ok, i thought it was just to test zip can figure out the same result

**Pull request closed without merging**